### DISCUSSION TITLE
팀 일정에 관련된 사항

### DISCUSSION THREAD

**Discussion by Topslug** [cau-swe/checkpoint-1#38] [General]
혹시 23일날 모두 시험이 끝난다고 한다면 23일 부터 25일 11:59까지 집중해서 마무리 작업 해도 되지 않을까요?
일정과 관련해서 서로 토의해보면 좋을 것 같습니다.

음, 그리고 저는 개인적으로 교수님께 26일까지 기한 연장을 요청 드려 보는 건 어떤지 다른 분들의 생각이 궁금합니다.
왜냐하면 중간고사 기간이 공식적으로는 26일까지 이기 때문입니다.

**Comment by Jummi10** [comment]
개강 전부터 정해져있었던 일정을 바꾸어달라고 요청드리는건 실례일 수도 있다고 생각이 듭니다
팀 일정을 잡는 사안은 깃허브보다 카톡에서 논의하거나 투표로 정하는게 더 효율적일 것 같네요

**Reply by Topslug** [reply]
아.. 그렇군요.. 제 생각이 짧았던 것 같습니다. 그러면 그렇게 하는 게 좋을 것 같습니다.
강의계획서상으로는 8주차에 일정이 있었는데 8주차가 26일까지라 그런 생각이 들었던 것 같습니다..
죄송합니다. 카톡으로 얘기해보는 게 좋을 것 같습니다.
